scripts/clean_dev.sh

```diff
@@ -0,0 +1,4 @@
+# grab any new gems
+bundle
+# delete, create and seed database
+rake db:drop db:create db:migrate db:seed
```

Comment:
**nitpick (blocking):** Use `bin/rails` instead of `rake`

Reply:
Fixed.